packages/invoices/src/OutboundShipment/DetailView/modals/utils.ts

```diff
@@ -40,8 +40,8 @@ export const sumAvailableQuantity = (
   draftOutboundLines: DraftOutboundLine[]
 ) => {
   const sum = draftOutboundLines.reduce(
-    (acc, { availableNumberOfPacks, packSize }) =>
-      acc + availableNumberOfPacks * packSize,
+    (acc, { stockLine, packSize }) =>
```

Comment:
updates from fields being on the nested `stockLine` rather than the draft line itself